sale_line_returned_qty_mrp/__manifest__.py

```diff
@@ -4,12 +4,12 @@
 {
     "name": "Sale Line Returned Qty Mrp",
     "summary": "Track returned quantity of sale order lines for BoM products.",
-    "version": "13.0.1.0.1",
+    "version": "14.0.1.0.0",
     "category": "Sales",
     "website": "https://github.com/OCA/stock-logistics-workflow",
     "author": "ForgeFlow, Odoo Community Association (OCA)",
     "license": "LGPL-3",
     "installable": True,
-    "depends": ["sale_line_returned_qty", "mrp"],
+    "depends": ["sale_line_returned_qty", "mrp", "stock_dropshipping"],
```

Comment:
Mmmh, I think this highlight :

- Either a wrong module name, that should reflect the dropshipping specific case.
- Either extra code about dropshipping cases that should maybe be externalized in another module.

@JasminSForgeFlow Could you clarify this ? Thanks

Reply:
Hi @rousseldenis,

I have added it because of a dropshipping test case.
Tried without dependency but test was failed

Thanks

Reply:
> I have added it because of a dropshipping test case.

You can't justify adding a dependency as a requirement for tests.



> Tried without dependency but test was failed

Of course, that will fail. But, as I said before :

- Either a wrong module name, that should reflect the dropshipping specific case.
- Either extra code about dropshipping cases that should maybe be externalized in another module.

@LoisRForgeFlow Any advice here ?

Reply:
Hi @JasminSForgeFlow 

You can get Odoo to evaluate a stock move as being a dropship without needing the stock_dropshipping module. It is in fact evaluated in stock_account without any dependency on stock_dropshipping:
https://github.com/odoo/odoo/blob/5d55841f580d7d193756218f192b49d5364bf2b0/addons/stock_account/models/stock_move.py#L115

Reply:
@JordiMForgeFlow but in this case, we have to add dependence on stock_account I think.
Can we handle it without it?

Reply:
@JasminSForgeFlow this module depends already on sale and on mrp, which implies that sale_mrp will also be installed.

sale_mrp depends on sale_stock, which depends also on stock_account. Therefore, the dependency is already there :)